### PR TITLE
Move dice to macros.html

### PR DESCRIPTION
**Pull request by agude**
This is the **ONLY** way I can figure out to avoid using `{{site.dice.pos}}`. Other things I've tried which do not work because of the Jekyll [Order of interpretation](https://jekyllrb.com/tutorials/orderofinterpretation/):

- Adding these macros to the _layout
- Adding them to index.md
- Adding them to 00-introduction.md
- Changing index.md to use `{{ chapter }}` or `{{ chapter.content }}` or `{{ chapter.output }}`

The only options I can see are:

- Include this at the top of chapters where we need it (a "do once" operation at file creation time)
- Live with `{{site.dice.pos}}`
- Write a custom Kramdown filter (and therefore write our own compilation and publish action on Github)
- Write javascript to parse something (I dislike this idea a lot)

**THIS PR IS FOR DISCUSSION** to demonstrate one solution, we can reject it freely!

### CODE REVIEW

**Comment by charles-uno** [comment]
this seems comparable to `{{site.dice.pos}}` to me. Content with either way

**Review by charles-uno — APPROVED** [review]
Love it

File: _code/make_dice_macros.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env python3
```
Comment (charles-uno) [inline _code/make_dice_macros.py:1]: Do we want to keep this file around?

Comment (agude) [reply _code/make_dice_macros.py:1]: Yes!

**Pull request merged**